### PR TITLE
pythonPackages.python-fontconfig: init at 0.5.1

### PR DESCRIPTION
**Pull request by ghost**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -5538,6 +5538,24 @@ in {
     };
   };
 
+  fontconfig = buildPythonPackage rec {
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: See the header of this file; the expression needs to be moved.

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -5538,6 +5538,24 @@ in {
...
+    name = "Python-fontconfig-0.5.1";
+
+    src = pkgs.fetchurl {
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: fetchPypi

**Review by FRidh — CHANGES_REQUESTED** [review]

**Comment by FRidh** [comment]
```
Ran 0 tests in 0.000s
```
No tests are found. Either the test runner cannot find tests and needs to be patched, or there simply are no tests in which case the tests need to be disabled. Do include a comment explaining why the tests are disabled.

**Comment by gnidorah** [comment]
@FRidh I've added the test, but it fails under chroot with `Fontconfig error: Cannot load default config file`

**Comment by FRidh** [comment]
You can set up a cache with
```
    fontsConf = pkgs.makeFontsCache {
      fontDirectories = [
        "${pkgs.ghostscript}/share/ghostscript/fonts"
      ];
```
and
```
export FONTCONFIG_FILE=${fontsConf}
```
or just disable the test.

**Comment by FRidh** [comment]
Good. Pushed b17ec549a14a4b3fb8cdfff914afddd65fee942e.

**Pull request closed without merging**